site/src/components/form.md

```diff
@@ -205,9 +205,16 @@ If you want to skip the basic validation on an input, apply a `data-skip-validat
 		</div>
 	</div>
 	<div class="field">
+		<legend class="field-label">Checkbox Field label</legend>
```

Comment:
per mdn, legends exist in the context of form fieldset, do we need the markup to reflect that? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/legend
```
<fieldset>
  <legend>Choose your favorite monster</legend>
```

Reply:
adjusted but kept class names